mne_features/tests/test_univariate.py

```diff
@@ -263,6 +263,8 @@ def test_powercurve_deviation():
     compute_powercurve_deviation and check whether the k1 and theta estimates
     are correct.
     """
+
+    np.random.seed(42)
```

Comment:
- I think it's a good habit not to set the global seed as you do here but, instead, use `rng = np.random.RandomState(seed=42)`. BTW, `rng` is already defined at the beginning of `test_univariate.py`.

- In `test_powercurve_deviation`, the variables `sfreq` and `data` shadow variables with the same names defined at the beginning of `test_univariate.py`. It would be more safe to use different names (or just use `sfreq` as it and use `data=sig.reshape(1, -1)` in  `compute_powercurve_deviation`).